pkg/authn/delegating.go

```diff
@@ -42,19 +42,20 @@ func NewDelegatingAuthenticator(client authenticationclient.TokenReviewInterface
 		p   *dynamiccertificates.DynamicFileCAContent
```

Comment:
does this still need to be declared up here?

Reply:
it's used in the return value